queries.sql

```diff
@@ -5,4 +5,63 @@ SELECT date_of_birth from animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered IS true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* Begin Transaction */
+BEGIN TRANSACTION;
+/* UPDATE species columns to 'unspecified' */
+UPDATE animals SET species = "unspecified";
+/* roll back the change */ 
+ROLLBACK;
```

Comment:
- [ ] Your transaction is missing a query to verify the change. Let's fix it
Hint: `SELECT species from animals; -- verify that change was made`